its/plugin/python-custom-rules-plugin/pom.xml

```diff
@@ -49,6 +54,7 @@
         <configuration>
           <pluginClass>org.sonar.samples.python.CustomPythonRulesPlugin</pluginClass>
           <requirePlugins>python:${project.version}</requirePlugins>
+          <sonarQubeMinVersion>${sonarQubeMinVersion}</sonarQubeMinVersion>
```

Comment:
why did you need to change this?

Reply:
Without this, the custom rules IT would crash because min SQ version seems to default to `sonar.version` when this property is missing, but the custom rules plugin IT seems to use an hardcoded 7.9 version (and I believed it was better to break backward compatibility with 7.9 explicitly once 8.9 is officially released as the LTS version).